spec/unit/entitlements/data/groups/calculated/yaml_spec.rb

```diff
@@ -35,6 +35,32 @@
     end
   end
 
+  describe "#schema_version" do
```

Comment:
can you add one more test where the version is `1.2` (no patch), should be caught by the regexp, but I'd like to see a test still :)

Reply:
@nobe4 I just added a test to cover this as well. It should be noted, that this test catches `1.2` and expects it to _fail_. This is because it is not valid [semver 2.0](https://semver.org/spec/v2.0.0.html).

Reply:
agreed 👍 thanks!